src/uuid/uuid.nim

```diff
@@ -2,8 +2,8 @@ import std/[strformat]
 import pkg/[uuids]
 import ".."/[logger]
 
-proc uuid*(n: Positive) =
-  ## Writes `n` version 4 UUIDs to stdout. Outputs only 1000 UUIDs if `n` is
+proc writeUuids(n: Positive) =
```

Comment:
Where are the uuids written to? Tiny suggestion:

```suggestion
proc outputUuids(n: Positive) =
```

Reply:
Done - see https://github.com/exercism/canonical-data-syncer/pull/106/commits/f78d103c60e9a0bb7316dfe6f20beab10f2a853a.